docs/products/range-tokens.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Range Tokens
+sidebar_label: Range Tokens
+---
+
+The range token enables a DAO or nascent project to use its native token as collateral to borrow funds. At maturity, if the debt is not paid, the range token holder is instead compensated with an equivalent amount of the collateral (the native token) using the settlement price of the native token to determine the number of tokens. 
+
+A range token can be viewed similarly to convertible debt. In the venture capital world, convertible debt allows startup companies to receive funding today without issuing equity upfront.
+
+## Example
+
+For example, if the native token is trading at $25 at settlement, a 100 USDC debt would be settled with 4 tokens (100 / 25). This is similar to a yield dollar or a CDP (collateralized debt position); however, there are a couple of added features.
+
+A range token seller (the DAO in this case) cannot be liquidated and therefore, the number of tokens given to the range token holder is capped. The range token holder is effectively short a put option and will have exposure to the downside of the native token below a certain price. To compensate the range token holder for taking on this “default” risk, the holder is rewarded a call option on the native token. A minimum number of native tokens is given to the range token holder no matter how much the native token rallies. The combination of the structure creates a tradeoff between the seller and the buyer.
```

Comment:
Think this section could use the numerical example you were starting in the first paragraph. Maybe also add the payout graphs we used in the articles - I found those useful conceptually.

Reply:
Yep good point. Added these!